packages/shared/src/components/MainLayout.tsx

```diff
@@ -46,9 +46,10 @@ const GoBackHeaderMobile = dynamic(
   { ssr: false },
 );
 
-const Sidebar = dynamic(
-  () => import(/* webpackChunkName: "sidebar" */ './sidebar/Sidebar'),
-  { ssr: false },
```

Comment:
Why did we end up removing the ssr: false?

Reply:
Doesn't feel like we need it actually here?
But maybe you or @capJavert know the replication case I could check to break it without SSR: false?